book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] I would recommend adding both Getters and Setters for `@rentals`

Reply:
resolved